### PR TITLE
Remove Generated Namespace Prefix in Generated Code

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description
We had the `Generated` prefix in generated code historically to prevent clashes with the C# SDK. Since we no longer rely on the C# SDK - removing this.

The longer we leave the prefix in, the harder it will be to change.
#### Tests
Nope
#### Documentation
Ran `grep -r "Generated" .` in `docs/` and changed any relevant references.
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: workers/unity/Assets/Playground/Scripts/MonoBehaviours/ProcessSpinnerColorChange.cs [hunk 1]
```diff
@@ -1,69 +1,69 @@
 using System.Collections.Generic;
-using Generated.Playground;
 using Improbable.Gdk.GameObjectRepresentation;
-using Playground;
 using UnityEngine;
-using Color = Generated.Playground.Color;
 
-public class ProcessSpinnerColorChange : MonoBehaviour
+namespace Playground.MonoBehaviours
```
Comment (zeroZshadow) [inline workers/unity/Assets/Playground/Scripts/MonoBehaviours/ProcessSpinnerColorChange.cs]: Why does this have a namespace now? But not the others?

Comment (jessicafalk) [reply workers/unity/Assets/Playground/Scripts/MonoBehaviours/ProcessSpinnerColorChange.cs]: isn't he fixing it for a multiple fileS? for some reason some files in the playground don't have a namespace :(

Comment (jamiebrynes7) [reply workers/unity/Assets/Playground/Scripts/MonoBehaviours/ProcessSpinnerColorChange.cs]: Yeah there were some missing namespaces

File: docs/content/gameobject/reading-and-writing-component-data.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ We provide code-generated Readers and Writers for interacting with SpatialOS com
 
 For every component defined in SpatialOS schema, we generate a pair of Readers and Writers within:
 
-* `Generated.<namespace of schema component>.<component name>.Requirable.Reader`
-* `Generated.<namespace of schema component>.<component name>.Requirable.Writer` 
+* `<namespace of schema component>.<component name>.Requirable.Reader`
```
Comment (JonasImprobable) [inline docs/content/gameobject/reading-and-writing-component-data.md]: Nice! Thanks! :)

**Pull request merged**